### PR TITLE
ng_ndp: code restructuring

### PR DESCRIPTION
**Pull request by authmillenon**
Some code restructuring to be in line with other code of similar format.

### CODE REVIEW

**Comment by PeterKietzmann** [comment]
Let the assignee decide :-) . I can confirm that the `ping6` command in native was not influenced by this PR

**Comment by PeterKietzmann** [comment]
@OlegHahm what do you think? I'm just trying to get some NSTF related PRs merged because I assume troubles when everyone tests with Haukes prepared TEST-branch but no one has an overview about the open and needed PRs.

File: sys/net/network_layer/ng_ndp/ng_ndp.c [hunk 1]
```diff
@@ -681,10 +679,10 @@ static void _send_nbr_sol(kernel_pid_t iface, ng_ipv6_addr_t *tgt,
         return;
     }
 
-    LL_PREPEND(pkt, hdr);
-
     ((ng_netif_hdr_t *)hdr->data)->if_pid = iface;
 
+    LL_PREPEND(pkt, hdr);
```
Comment (OlegHahm) [inline sys/net/network_layer/ng_ndp/ng_ndp.c]: What's the effect of this swapping of lines?

Comment (authmillenon) [inline sys/net/network_layer/ng_ndp/ng_ndp.c]: Should be none. It just looked weird to me to manipulate the headers content after it was prepended to the packet. So it's basically just a cosmetic change (as most of this PR is), ordering the LOC a little bit more thematically.

**Comment by OlegHahm** [comment]
ACK

**Comment by PeterKietzmann** [comment]
Travis is green

**Pull request merged**